pychatml/anthropic_converter.py

```diff
@@ -0,0 +1,38 @@
+import re
+from typing import List, Dict, Any
+from anthropic import AI_PROMPT, HUMAN_PROMPT
+
+from pychatml.chatml_converter import ChatMLConverter
```

Comment:
I think chatml converter is a weird  name both for the class and the package. Maybe just abstract and abstract converter